tests/unit/remote/ssh/test_connection.py

```diff
@@ -64,3 +76,22 @@ def test_walk(tmp_path, ssh):
             paths.add(posixpath.join(root, entry))
 
     assert paths == expected
+
+@pytest.mark.skipif(platform.system() != "Darwin",
+                    reason="Only works on OSX (because of APFS)")
+def test_reflink(chdir_tmp, foo, ssh):
+    ssh.reflink("foo", "bar")
+    assert System.is_reflink(bar_path)
```

Comment:
There is no such function. We've discussed that there is no good way to distinguish copy and reflink, right? So just check that bar_path exists and is not a symlink or a hardlink, it will more than enough.

Reply:
Right :+1: